File: Pythagoras.jl
"""
Author: Yoshiyosi
"""
function pythag(a,b)
    return √(a**2+b**2)
end

function pythag2(a,b)
    if a>=b
        return √(a**2-b**2)
    end
    if a<b
        return √(b**2-a**2)
    end
end

println(pythag(3,4))
println(pythag(√(2),√(2)))
println(pythag2(5,4))
println(pythag2(4,5))

File: Pythagoras.py
# -*- coding: utf-8 -*-

"""
Author: Yosiyoshi
"""
import math

def pythag(a,b):
    c2=a**2+b**2
    c=math.sqrt(c2)
    return c
    
def pythag2(a,b):
    if a>=b:
        c2=a**2-b**2
        c=math.sqrt(c2)
        return c
    if a<b:
        c2=b**2-a**2
        c=math.sqrt(c2)
        return c

print(pythag(3,4))
print(pythag(math.sqrt(2),math.sqrt(2)))
print(pythag2(5,4))
print(pythag2(4,5))